packages/nx/bin/post-install.ts

```diff
@@ -3,10 +3,13 @@ import { workspaceRoot } from '../src/utils/workspace-root';
 import { fileExists } from '../src/utils/fileutils';
 import { join } from 'path';
 import { daemonClient } from '../src/daemon/client/client';
+import { assertSupportedPlatform } from '../src/native/assert-supported-platform';
 
 (async () => {
   try {
```

Comment:
I think we need to move this `try catch` into the `if` block. Otherwise, the error thrown by `assertSupportedPlatform` will not be shown.

Reply:
It is shown, I tested it by publishing a local copy and doing `yarn --no-optional`

Reply:
Oh because we have `process.exit(1)` in there. Got it.